clients/web/src/stylesheets/layout/global.styl

```diff
@@ -111,6 +111,20 @@ textarea
 /* Stuff that applies across the top level object lists */
 .g-top-level-list-title-container
   display inline-block
+  overflow hidden
+  white-space nowrap
+  width calc(100% - 350px)
```

Comment:
Makes sense. For future reference, see https://www.w3.org/TR/css3-mediaqueries/#media0